supabase/functions/_backend/public/organization/members/post.ts

```
@@ -14,27 +16,85 @@ const inviteBodySchema = z.object({
     'write',
     'admin',
     'super_admin',
+    'org_member',
+    'org_billing_admin',
+    'org_admin',
+    'org_super_admin',
   ]),
 })
 
-export async function post(c: Context, bodyRaw: any, apikey: Database['public']['Tables']['apikeys']['Row']) {
+const legacyInviteRoles = ['read', 'upload', 'write', 'admin', 'super_admin'] as const
```

ESLint

[error] 'legacyInviteRoles' is assigned a value but only used as a type. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)